src/components/LoginTest.jsx

```diff
@@ -1,13 +1,54 @@
 import React, { useState } from 'react';
-import { logInWithEmailAndPassword } from '../utils/auth_utils';
+import { instanceOf } from 'prop-types';
+import { cookieConfig, cookieKeys, Cookies, withCookies } from '../utils/cookie_utils';
+import {
+  auth,
+  logInWithEmailAndPassword,
+  useNavigate,
+  signInWithGoogle,
+} from '../utils/auth_utils';
 
-const LoginTest = () => {
+const LoginTest = ({ cookies }) => {
+  const navigate = useNavigate();
```

Comment:
Add a new `errorMessage` state to store/display any errors during account creation. If an error is thrown during `logInWithEmailAndPassword`, it should be rendered on the page.